docker/build-remote.sh

```diff
@@ -25,11 +25,15 @@ cp -R bin default ${WORK_DIR}/presto-server-${PRESTO_VERSION}
 curl -o ${WORK_DIR}/presto-cli-${PRESTO_VERSION}-executable.jar ${CLIENT_LOCATION}
 chmod +x ${WORK_DIR}/presto-cli-${PRESTO_VERSION}-executable.jar
 
-docker build ${WORK_DIR} --pull -f Dockerfile -t "presto:${PRESTO_VERSION}" --build-arg "PRESTO_VERSION=${PRESTO_VERSION}"
+# Loading multi-arch images is not supported
+DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build ${WORK_DIR} --platform linux/amd64  -f Dockerfile --build-arg "PRESTO_VERSION=${PRESTO_VERSION}" -t "presto:${PRESTO_VERSION}" --load
 
 rm -r ${WORK_DIR}
 
 # Source common testing functions
 . container-test.sh
 
 test_container "presto:${PRESTO_VERSION}"
+
+# Push multi-arch images supporting amd64 and arm64 architecture.
+DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build ${WORK_DIR} --platform linux/amd64,linux/arm64/v8 -f Dockerfile --build-arg "PRESTO_VERSION=${PRESTO_VERSION}" -t "presto:${PRESTO_VERSION}" --push
```

Comment:
Hi, @Lewuathe What about using `aarch64` instead of `arm64/v8`?  I found presto use `aarch64` in source code [here](https://github.com/prestosql/presto/blob/463f0e142e2d0a607cae5e57880cea44f1c793eb/presto-main/src/main/java/io/prestosql/server/PrestoSystemRequirements.java#L73) and looks like `aarch64` is more popular.

Reply:
As far as I know, there is no `aarch64` parameter in the Docker builder. We need to specify `arm64/v8` as a `TARGETPLATFORM` parameter. But `arm64/v8` and `aarch64` are basically the same thing, IIUC.

```
$ docker buildx ls
NAME/NODE            DRIVER/ENDPOINT             STATUS  PLATFORMS
default              docker
  default            default                     running linux/amd64, linux/arm64, linux/ppc64le, linux/s390x, linux/386, linux/arm/v7, linux/arm/v6

Reply:
We should make this a separate script like `push-image.sh`, so that we can build images separately from pushing them.

Reply:
I'm confused what `--push` does. It says it pushes it to a repository, but which one? We don't specify `prestosql` anywhere as the Docker Hub repository. Is there a way to build the ARM64 image without pushing?

Reply:
I think we should prepare the separate script `push-image.sh` so that we can push both images by `--push` option to `prestosql` orgniazation. But I do not have a way to check pushing to `prestosql` organization properly. How do you usually push Docker images to `prestosql`?